src/Arduino_WiFiConnectionHandler.cpp

```diff
@@ -175,7 +175,12 @@ void WiFiConnectionHandler::update() {
         }
         break;
       case NetworkConnectionState::GETTIME: {
-
+        Debug.print(DBG_VERBOSE, "NetworkConnectionState::GETTIME");
+#if defined(ARDUINO_ESP8266_ESP12) || defined(ARDUINO_ARCH_ESP32) || defined(ESP8266)
+        configTime(0, 0, "pool.ntp.org", "time.nist.gov");
```

Comment:
We do also have our own NTP server at time.arduino.cc in case you want to include it as well.

Reply:
Good catch :+1: I've added the Arduino NTP server in 97b2238d70988818455c1587e1cbe833c32f9caf .